samples/net/cloud/aws_mqtt/src/certificates.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * openssl x509 -in <filename>.pem -inform pem -noout -C
+ */
+
+#error "Fill this before generating the build"
+
+unsigned char amazon_certificate[] = {
```

Comment:
I would prefer that we do things here similar way as in other samples where some file needs to be embedded into the binary. So please write this as
```
unsigned char amazon_certificate[] = {
#include "amazon.cert"
}
```
if the cert is in ascii format. If you have the cert in binary format (DER), there is automatic way to create a file that can be included here (search generate_inc_file in networking samples for an example).
The point here is that the amazon.cert file is not put to version control and this will not compile if user does not provide the file. Some nice error message can be generated of course.